keen/tests/client_tests.py

```diff
@@ -8,20 +8,37 @@
 import keen
 from keen.client import KeenClient
 from keen.tests.base_test_case import BaseTestCase
+from mock import patch, MagicMock
 import sys
 
 __author__ = 'dkador'
 
 
+class MockedRequest(object):
+    def __init__(self, status_code, json_response):
+        self.status_code = status_code
+        self.json_response = json_response
+
+    def json(self):
+        return {"result": self.json_response}
+
+
+class MockedFailedRequest(MockedRequest):
+    def json(self):
+        return self.json_response
+
+
+@patch("requests.Session.post")
 class ClientTests(BaseTestCase):
```

Comment:
Maybe add the standard `201` and `200` `MockedRequest` objects up here like you did in the `QueryTests` object? `SINGLE_ADD_RESPONSE` and `MULTI_ADD_RESPONSE` or something?